Marlin/Marlin_main.cpp

```diff
@@ -2182,6 +2182,11 @@ static void clean_up_after_endstop_or_probe_move() {
     #if ENABLED(BLTOUCH)
       if (set_bltouch_deployed(true)) return true;
     #endif
```

Comment:
The BLTouch will throw an error if it is deployed for too long. If the machine sits waiting for the bed to heat, all BLTouch probes and clones will throw an error and the errored-out BLTouch probe will then be pushed into the bed, crushing the pin and damaging the probe.

Reply:
I have now moved the code so that it waits before the BL touch is deployed. This should eliminate that error.

Reply:
The problem is not only BL touch related but to all kinds of probes. Why should the solution depend on BL touch only?

Reply:
Sorry. Did not read the code, just the comment.